static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -361,34 +361,16 @@ $gutter-large: 55px;
         margin-right: $gutter-large + ($veggie-burger-medium / 2);
     }
 
+    @include mq($mobile-medium) {
+        font-size: 23px;
```

Comment:
If you look at making the type reduce/increase by percentage across the board then we can have a rule for this happening elsewhere! Refer to our conversation in person! Otherwise 👍